sites/website/pages/en/help.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require("react");
+
+const CompLibrary = require("../../core/CompLibrary.js");
+
+const Container = CompLibrary.Container;
+const GridBlock = CompLibrary.GridBlock;
+
+const siteConfig = require(`${process.cwd()}/siteConfig.js`);
+
+function docUrl(doc, language) {
+    return `${siteConfig.baseUrl}docs/${language ? `${language}/` : ""}${doc}`;
+}
+
+class Help extends React.Component {
+    render() {
+        const language = this.props.language || "";
+        const supportLinks = [
+            {
+                content: `Learn more using the [documentation on this site.](${docUrl(
+                    "doc1.html",
+                    language
+                )})`,
+                title: "Browse Docs",
+            },
+            {
+                content: "Ask questions about the documentation and project",
+                title: "Join the community",
+            },
+            {
+                content: "Find out what's new with this project",
+                title: "Stay up to date",
+            },
+        ];
+
+        return (
+            <div className="docMainWrapper wrapper">
+                <Container className="mainContainer documentContainer postContainer">
+                    <div className="post">
+                        <header className="postHeader">
+                            <h1>Need help?</h1>
+                        </header>
+                        <p>This project is maintained by a dedicated group of people.</p>
+                        <GridBlock contents={supportLinks} layout="threeColumn" />
+                    </div>
+                </Container>
+            </div>
```

Comment:
Should these three be moved?

Reply:
these were moved but unfortunately git had issues treating this as a move.

Reply:
`Help.js` and `Users.js` could be removed.  Index.js cannot.  It's in use.  `/website/pages` is where custom React pages go for Docusaurus to compiles Markdown, and marketing content into a website.  This occur after running the TypeDocs utility to generate additional MarkDown files.

Reply:
That said.  I would think for consistency any Docusaurus cleanup would be outside the scope of this reorganization.